model/perf_data.go

```diff
@@ -320,25 +319,27 @@ type PerformanceMetricSummary struct {
 // reporting that this application does across test values.
 type PerformanceTimeSeries []PerformancePoint
 
-// Statistics converts a a series into an intermediate format that we
-// can use to calculate means/totals/etc.
-func (ts PerformanceTimeSeries) Statistics() PerformanceStatistics {
-	out := PerformanceStatistics{
+// statistics converts a a series into an intermediate format that we
+// can use to calculate means/totals/etc to create default rollups
+func (ts PerformanceTimeSeries) statistics() (performanceStatistics, error) {
```

Comment:
probably worth returning a pointer here to avoid needless copying and also being able to return nil.